test/Libraries/DynamoPythonTests/PythonEvalTestsWithLibraries.cs

```diff
@@ -154,19 +158,16 @@ import clr
 r = range(0, 10, 2)
 # Python range => .NET array - Works in both
 a = DummyCollection.MakeArray(r)
-# Python range => .NET IList - Does not work in CPython
```

Comment:
it does make me wonder if this should work - but through an exception - so that users don't expect all future engines to support all conversions...

IE when ironPython 3 exists, maybe we only want to support a limited set of conversions.

Reply:
That one came for free because ranges can be iterated over and it just works. Iron Python 3 might add support for this conversion, it makes sense if they do it. I guess I see your point, but I don't think we should claim that at some point the engines will converge. I guess people will either stick to one or learn to live with their differences.